types/pem/index.d.ts

```diff
@@ -135,6 +136,12 @@ export interface CertificateSubjectReadResult {
     emailAddress: string;
 }
 
+export interface Pkcs12ReadResult {
+    key: string;
+    cert: string;
+    ca: any[];
```

Comment:
According to the [source](https://github.com/andris9/pem/blob/4f36e55/lib/pem.js#L1221), `ca` is an array of `string`. Can we change this `any[]` to `string[]`?

Reply:
thanks, sure changed